(DOCSP-16456): Added new field. (#557)
mongodb/mongodb-kubernetes-operator: docs/deploy-configure.md

   | Key | Type | Description | Required? |
   |----|----|----|----|

>    | `spec.security.authentication.ignoreUnknownUsers` | boolean | 
>    Flag that indicates whether you can add users that do not exist 
>    in the `MongoDBCommunity` resource. If omitted, defaults to 
>    `true`. | No | 

Reviewer: - remove the line breaks. github's markdown renderer handles these differently than our docs tooling. Click the kebab menu for this file on the `Files Changed` tab for this PR, then `View File` to see how this renders.

- do not -> don't ([per style guide)](https://docs.mongodb.com/meta/style-guide/style/contractions/)

```suggestion
   | `spec.security.authentication.ignoreUnknownUsers` | boolean | Flag that indicates whether you can add users that don't exist in the `MongoDBCommunity` resource. If omitted, defaults to `true`. | No | 
```

Reviewer: also consider changing `MongoDBCommunity` resource to `MongoDB resource definition`. This term is used elsewhere in the docs.

Reviewer 2: we renamed from `MongoDB` to `MongoDBCommunity` to avoid confusion with the enterprise `MongoDB` resource. I think if we are explicitly referring to the Kubernetes CR we can use MongoDBCommunity, what do you think?

Reviewer: sounds good to me. We should make this change elsewhere in the docs, too.

Author: I'll open another ticket about the renaming.
